.vscode/launch.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Maybe I have to delete this; it is the debug launchers I use for Visual Studio Code. Would you like to keep it or delete it?

Reply:
@Andrewgdewar you're the only other person that uses vscode i think, your thoughts?